docs/api/tutorials/forms.md

```diff
@@ -1,13 +1,13 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Documentation Forms
+# Compliance Forms
 
-## Why Would You Use Documentation Forms?
+## Why Would You Use Compliance Forms?
 
-Documentation Forms are a way for end-users to fill out all mandatory attributes associated with a data asset. The form will be dynamically generated based on the definitions provided by administrators and stewards and matching rules.
+Compliance Forms are a way for end-users to fill out all mandatory attributes associated with a data asset. The form will be dynamically generated based on the definitions provided by administrators and stewards and matching rules.
```

Comment:
Compliance Forms are not about "end-users" filling out requirements- end users don't care!
They'd prefer to NOT be filling out requirements.

Instead, Compliance Forms are a way to crowd source critical governance and compliance attributes across your data assets AT SCALE. Our target end user if a Data Governance/Compliance Leader, not the end user. We need to ensure we are always speaking about empowering them, first and foremost. They are the persona we need to click with on this

Reply:
Updating to use the copy you suggested here https://github.com/datahub-project/datahub/pull/11936/files#r1866766843